recipes-ni/ni-systemreplication/files/nisystemreplication

```diff
@@ -74,7 +74,7 @@ is_image_valid () {
 }
 
 image_get () {
-	if ! mount -o remount,rw $NIRECOVERY_MOUNTPOINT; then
+	if ! mount -o remount,rw,async $NIRECOVERY_MOUNTPOINT; then
```

Comment:
We should add an explicit `sync` call after the image_get operation for safety; right?

Reply:
Done